src/components/dockerRegistry/Docker.tsx

```diff
@@ -183,7 +205,7 @@ function CollapsedList({
                 <div className="flex left">
                     <List.Title
                         style={{ color: !id && !collapsed ? 'var(--N900)' : '' }}
-                        title={id || 'Add Container Registry'}
```

Comment:
is it expected?

Reply:
expected as now we will also configure OCI registry here